### PR TITLE
feat: Border radius and new shades

### PR DESCRIPTION
**Pull request by albinAppsmith**
## Description

> Please include a summary of the changes and which issue has been fixed. Please also include relevant motivation
> and context. List any dependencies that are required for this change.

Fixes # (issue)

> if no issue exists, please create an issue and ask the maintainers about this first

## Type of change

> Please delete options that are not relevant.

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to not work as expected)
- This change requires a documentation update

## How Has This Been Tested?

> Please describe the tests that you ran to verify your changes. Provide instructions, so we can reproduce.
> Please also list any relevant details for your test configuration.

- Test A
- Test B

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by tanvibhakta — APPROVED** [review]

File: packages/design-system/src/Dropdown/styles.css [hunk 1]
```diff
@@ -1,3 +1,26 @@
+@import url("../commonStyles/popover.css");
+
+:root {
```
Comment (tanvibhakta) [inline packages/design-system/src/Dropdown/styles.css:3]: Same for this styles.css file

**Review by tanvibhakta — COMMENTED** [review]
In general, I see two issues in this PR that might not scale for us: 
1. There are a bunch of style.css files importing things that we can directly import in the file instead. Let's minimize those files as much as possible to avoid having a bloated css architecture.
2. The theme variables are directly being assigned to the css. When we were migrating these components, we decided to create component tokens so that it could be easy to update them. I see the component tokens being usedi some places and not in others, and it feel haphazard. Can we go back to using the tokens everywhere? https://www.notion.so/appsmith/Refactoring-Roadmap-359cbcf8e0384e819232813670859904#fc96d40d124e413699a097e0c7f12b79

File: packages/design-system/src/themes/default/index.css [hunk 2]
```diff
@@ -103,4 +103,44 @@
   /** Uneditable Text Field */
   --ads-uneditable-text-field-label-text-color: var(--ads-color-black-750);
 
+
+
+  /* Variables V2 */
+
+  /** Foreground/text **/
+  --ads-color-fg-emphasis-plus: var(--ads-color-gray-800);
```
Comment (tanvibhakta) [inline packages/design-system/src/themes/default/index.css:111]: re: fg/bg - please let us use full forms of things as much as we can. It improves readability and in the era of modern IDEs makes no discernible difference in typing time.

**Review by tanvibhakta — CHANGES_REQUESTED** [review]

**Comment by albinAppsmith** [comment]
Closing this PR.

**Pull request closed without merging**